packages/zowe-explorer/i18n/sample/package.i18n.json

```diff
@@ -88,6 +88,7 @@
   "paste": "Paste Member",
   "hMigrateDataSet": "Migrate Data Set",
   "hRecallDataSet": "Recall Data Set",
+  "showImperativeErrorDetails": "Show More Information",
```

Comment:
Would it be clearer to say "Show Error Details" since this menu item only applies when there is error info to show?